core/computed/entity-classification.js

```diff
@@ -20,11 +20,13 @@ class EntityClassification {
    */
   static makeUpAnEntity(entityCache, url) {
     if (!UrlUtils.isValid(url)) return;
-    // We can make up an entity only for those URLs with a valid domain attached.
-    // So we further restrict from allowed URLs to (http/https).
-    if (!Util.createOrReturnURL(url).protocol.startsWith('http')) return;
 
-    const rootDomain = Util.getRootDomain(url);
+    const parsedUrl = Util.createOrReturnURL(url);
+    const isChromeExtension = parsedUrl.protocol.startsWith('chrome-extension');
+    // Make up an entity only for valid http/https URLs and Chrome extensions.
+    if (!isChromeExtension && !parsedUrl.protocol.startsWith('http')) return;
+
+    const rootDomain = isChromeExtension ? 'Chrome Extensions' : Util.getRootDomain(url);
```

Comment:
we should probably translate this string

Reply:
Not sure if translating it here would be the correct approach, since this makes it to the JSON, so querying by a specific entity might become difficult. One way we could translate it is to leave it as it is here, or define it as a `CHROME_EXTENSIONS` string and during reporting run through each and match to an available translation. Wdyt?

Reply:
> CHROME_EXTENSIONS string and during reporting run through each and match to an available translation. Wdyt?

I think that's how our current translating pipeline works. The untranslated (english) version of this string will be translated by the time it is placed in the LHR, at least that's what I would hope. The report renderer would not do any translation of it's own.

Not sure if this would happen automatically, so if it's too much work I'm fine leaving it alone.